src/parcsr_ls/par_ilu_solve.c

```diff
@@ -1164,577 +1201,32 @@ hypre_ILUSolveLURAS(hypre_ParCSRMatrix *A,
          /* diagonal scaling (contribution from D. Note: D is stored as its inverse) */
          uext[idx] *= D[i];
       }
-
-   }
-   /* Update solution */
-   hypre_ParVectorAxpy(beta, utemp, u);
-
-   return hypre_error_flag;
-}
-
-#if defined(HYPRE_USING_CUDA) && defined(HYPRE_USING_CUSPARSE)
-
-/* Permutation function (for GPU version, can just call thrust)
- * option 00: perm integer array
- * option 01: rperm integer array
- * option 10: perm real array
- * option 11: rperm real array
- * */
-HYPRE_Int
-hypre_ILUSeqVectorPerm(void      *vectori,
```

Comment:
Has this been completely removed or renamed?

Reply:
Is `hypre_ILUSolveCusparseLU` removed?